src/umpire/device_allocator_helper.hpp

```diff
@@ -72,22 +72,26 @@ __host__ void destroy_device_allocator();
  * synced up and pointing to each other.
  */
 #if defined(UMPIRE_ENABLE_CUDA)
-#define UMPIRE_SET_UP_DEVICE_ALLOCATORS()                                                                            \
-  {                                                                                                                  \
-    if (umpire::macro_tracking == 0) {                                                                               \
-      UMPIRE_LOG(Debug, "Calling cudaMemcpyToSymbol DeviceAllocator macro.");                                        \
-      cudaMemcpyToSymbol(umpire::UMPIRE_DEV_ALLOCS, &umpire::UMPIRE_DEV_ALLOCS_h, sizeof(umpire::DeviceAllocator*)); \
-    }                                                                                                                \
-    umpire::macro_tracking = 1;                                                                                      \
+#define UMPIRE_SET_UP_DEVICE_ALLOCATORS()                                                           \
+  {                                                                                                 \
+    if (umpire::macro_tracking == 0) {                                                              \
+      UMPIRE_LOG(Debug, "Calling cudaMemcpyToSymbol DeviceAllocator macro.");                       \
+      cudaError_t err = cudaMemcpyToSymbol(umpire::UMPIRE_DEV_ALLOCS, &umpire::UMPIRE_DEV_ALLOCS_h, \
+                                           sizeof(umpire::DeviceAllocator*));                       \
+      UMPIRE_USE_VAR(err);                                                                          \
```

Comment:
Should this error be reported as well?

Reply:
Because it's in a macro, it acts all weird with the UMPIRE_ERROR thing. I forget the exact error I got, but it did not like having that in there. I could try again in a sec and note what the error message is...

Reply:
We will address this in the issue #975